[2022-10-02 00:01:49,509] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.store_fct_table scheduled__2022-10-01T00:00:00+00:00 [queued]>
[2022-10-02 00:01:49,652] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: main_dag.store_fct_table scheduled__2022-10-01T00:00:00+00:00 [queued]>
[2022-10-02 00:01:49,689] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-10-02 00:01:49,721] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-10-02 00:01:49,736] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-10-02 00:01:49,869] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): store_fct_table> on 2022-10-01 00:00:00+00:00
[2022-10-02 00:01:49,928] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'main_dag', 'store_fct_table', 'scheduled__2022-10-01T00:00:00+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmp3backqsi', '--error-file', '/tmp/tmpzelx1un5']
[2022-10-02 00:01:49,916] {standard_task_runner.py:52} INFO - Started process 5263 to run task
[2022-10-02 00:01:49,976] {standard_task_runner.py:80} INFO - Job 121: Subtask store_fct_table
[2022-10-02 00:01:51,409] {task_command.py:369} INFO - Running <TaskInstance: main_dag.store_fct_table scheduled__2022-10-01T00:00:00+00:00 [running]> on host 8fa0dcc1c8f5
[2022-10-02 00:02:01,677] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=main_dag
AIRFLOW_CTX_TASK_ID=store_fct_table
AIRFLOW_CTX_EXECUTION_DATE=2022-10-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-10-01T00:00:00+00:00
[2022-10-02 00:02:01,691] {postgres.py:127} INFO - Running copy expert: COPY fct_table FROM stdin WITH DELIMITER as ',' CSV HEADER, filename: df_fct_table.csv
[2022-10-02 00:02:01,807] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-10-02 00:02:01,917] {python.py:173} INFO - Done. Returned value was: None
[2022-10-02 00:02:01,992] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=main_dag, task_id=store_fct_table, execution_date=20221001T000000, start_date=20221002T000149, end_date=20221002T000201
[2022-10-02 00:02:02,097] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-10-02 00:02:02,143] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
